packages/blade/src/components/SideNav/_decisions/decisions.md

```diff
@@ -0,0 +1,623 @@
+
+<table>
+<tr>
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNav>
+  <SideNavSection 
+    title="OFFERINGS SECTION" 
+    maxVisibleItems={3}
+    onToggleVisibleItems={(isExpanded) => {
+      console.log("When +13 More is clicked")
+    }}
+  >
+    {/* All SideNavLink items */}
+  </SideNavSection>
+</SideNav>
+```
+
+</td>
+
+<td>
+<img width="200px" src="navsection-eg.png" />
+</td>
+</tr>
+</table>
+
+### SideNavItem
+
+A generic non-link item that gives you leading, trailing slots to render the components as per your usecase.
+
+> [!Note]
+>
+> SideNavItem cannot be active, cannot be link, and cannot be button. It can be static item with something interaction in trailing slot
+
+| **Props** | **Description**                                                                      | **Type**     | **Default Value** |
+| --------- | ------------------------------------------------------------------------------------ | ------------ | ----------------- |
+| title     | title of SideNavItem                                                                 | string       |                   |
+| as        | as prop for passing render element. Use `label` if trailing has some input component | div          |                   |
+| leading   | leading slot. Render indicator / icon here                                           |              |                   |
+| trailing  | Trailing Slot of Item. Render Switch, or any other blade component here              | JSX          |                   |
+| tooltip   | Object with props that are forwarded to tooltip                                      | TooltipProps | undefined         |
+
+For Example of SideNavItem, check SideNavFooter example below.
+
+### SideNavFooter
+
+| **Props** | **Description** | **Type** | **Default Value** |
+| --------- | --------------- | -------- | ----------------- |
+| children  | Children slot   | JSX      |                   |
+
+<table>
+<tr>
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNav>
+  {/* ... Other Items */}
+  <SideNavFooter>
+    <SideNavItem
+      as="label"
+      title="Test Mode"
+      leading={
+        <Indicator
+          color={
+            isTestModeActive 
+            ? 'notice' 
+            : 'positive'
+          }
+          emphasis="intense"
+          accessibilityLabel=""
+        />
+      }
+      backgroundColor={
+        isTestModeActive 
+        ? `feedback.background.notice.subtle` 
+        : undefined
+      }
+      trailing={
+        <Switch
+          accessibilityLabel=""
+          size="small"
+          isChecked={isTestModeActive}
+          onChange={({ isChecked }) => {
+            setIsTestModeActive(isChecked);
+          }}
+        />
+      }
+    />
+    <SideNavLink 
+      as={NavLink}
+      icon={GearIcon}
+      href="/settings" 
+      title="Settings" 
+    >
+      {/* L2 SideNavLink items */}
+    </SideNavLink>
+  </SideNavFooter>
```

Comment:
How will this actually work? 

Because SideNavLink will have it's own styling (hover, bg etc) and you are putting it inside SideNavItem which will also have it's own styling.